frontend/src/metabase/ui/components/overlays/Popover/Popover.styled.tsx

```diff
@@ -30,6 +30,12 @@ export const getPopoverOverrides = (): MantineThemeOverride["components"] => ({
         const target = event.target as HTMLElement;
         target.setAttribute("data-ignore-outside-clicks", "true");
       },
+      onTouchStartCapture: (event: SyntheticEvent) => {
```

Comment:
We probably need to also handle this event in `frontend/src/metabase/ui/components/overlays/Menu/MenuItem/MenuItem.tsx`